stock_reception_screen/__manifest__.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+{
+    "name": "Stock - Reception screen",
+    "summary": "Dedicated screen to receive/scan goods.",
+    "version": "12.0.1.0.0",
+    "category": "Stock",
+    "license": "AGPL-3",
+    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "depends": [
+        "stock",
+        "product_expiry",
+        # OCA/product-attribute
+        "product_packaging_dimension",
+        # OCA/stock-logistics-workflow
+        "stock_quant_package_dimension",
+        # OCA/wms
+        "stock_location_storage_type_strategy",
```

Comment:
```suggestion
        "stock_storage_type_putaway_strategy",
```

Reply:
Or maybe `stock_storage_type` might be enough as I'm not sure it requires the putaway machinery.

Reply:
No we need `stock_storage_type_putaway_strategy` as this module is using the `stock.quant.package.package_storage_type_id` field define in your module.